components/skus/common/skus_sdk.mojom

```diff
@@ -0,0 +1,13 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+module skus.mojom;
+
+interface SdkService {
```

Comment:
Specifically, it's wrapping the SKU SDK as a service. Definitely could be a better name for it though
cc: @evq @bridiver

Reply:
It's not wrapping anything, it is defining the interface for the skus service and the implementation will be Impl

Reply:
The Software Development Kit Service? It should be SkusService as @goodov suggested.

Reply:
@bridiver it would be used like `skus::SdkService()` (as there may be other SKU services down the road: pay with bat, etc). For now, it's the only service; I'll rename it

Reply:
Renamed in https://github.com/brave/brave-core/pull/10847/commits/db48260dd1e9b819e4d991e99f8dcb3c31ca364c